java/src/org/openqa/selenium/HasDownloads.java

```diff
@@ -27,7 +27,13 @@ public interface HasDownloads {
   /**
    * Requires downloads to be enabled.
    *
-   * <p>TODO: Create an example in the documentation and provide a link to it.
+   * <p>Example usage:
+   *
+   * <pre>{@code
+   * ChromeOptions options = new ChromeOptions();
+   * options.setCapability("se:downloadsEnabled", true);
+   * driver.requireDownloadsEnabled(options);
+   * }</pre>
```

Comment:
What was meant by "Create an example in the documentation and provide a link to it.", was to create a code example in the documentation repository and point to it.

So the JavaDoc should have a phrase pointing to https://www.selenium.dev/documentation/webdriver/drivers/remote_webdriver/#downloads.

Reply:
@diemol thanks for your advice 🙏
I'll finish it later this week